exercises/numbers/isMultipleOf/isMultipleOf.js

```diff
@@ -13,7 +13,18 @@
  * @returns {boolean} True if n is a multiple of d and false otherwise
  */
 function isMultipleOf(n, d) {
-  return _____;
+
+  if (n === 0){
```

Comment:
Technically `0` is  a multiple of _everything_. You'll notice if you assume that, all the logic below simplifies.

As above, no need for the conditional, the following works:

```js
function isMultipleOf(n, d) {
  return n % d === 0;
}
```

Reply:
This logic makes sense, but when I do the 'Checking 0' in main, three of the lines are evaluating incorrectly to true, when they should be false.

Reply:
Good catch! I must've gotten the arguments backwards.

While `0` is a multiple of every number, only `0` is a multiple of `0`.

So:

1. `isMultipleOf(0, x)` should always return `true`
1. `isMultipleOf(x, 0)` should return `false` if `x !== 0` and `true` if `x === 0`.

Reply:
I guess `5 % 0` in JavaScript evaluates to `NaN` (a special value that means "not a number", used to represent things like division by `0`). This would mean that `0 % 0 === 0` would evaluate to `false`. So I guess the logic would have to be something like:

```javascript
functions isMultipleOf(n, d) {
  if (n === 0) {
    return true;
  }

  return n % d === 0;
}
```